### PR TITLE
Move the views check into the widget check.

### PR DESCRIPTION
**Pull request by Camwyn**
Ensure that if the constant is defined but v2 isn't enabled we get the v1 widget.

[TEC-3593]

constant false, v1: http://p.tri.be/ltkVZa
constant false, v2: http://p.tri.be/EIFcrj

constant true, v1: http://p.tri.be/MILjxR
constant true, v2: http://p.tri.be/cirDgo

no constant, v1: http://p.tri.be/ggp0O0
no constant, v2: http://p.tri.be/1f20EU

[TEC-3593]: https://moderntribe.atlassian.net/browse/TEC-3593

### CODE REVIEW

File: tests/views_integration/Tribe/Events/Views/V2/ViewTest.php [hunk 1]
```diff
@@ -79,7 +79,7 @@ public function should_return_an_instance_of_a_specified_view_if_provided() {
 	 *
 	 * @test
 	 */
-	public function should_print_a_view_html_on_the_page_when_caling_send_html() {
+	public function should_print_a_view_html_on_the_page_when_calling_send_html() {
```
Comment (Camwyn) [inline tests/views_integration/Tribe/Events/Views/V2/ViewTest.php:82]: typo fix

File: tests/views_wpunit/Tribe/Events/Views/V2/Widget_BootstrapTest.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+		$this->assertTrue( tribe_events_widgets_v2_is_enabled() );
+	}
+
+	/**
+	 * @test
+	 *
+	 * It should return false when v2 view is disabled.
+	 */
+	public function it_should_return_false_when_view_disabled() {
```
Comment (Camwyn) [inline tests/views_wpunit/Tribe/Events/Views/V2/Widget_BootstrapTest.php:85]: everything from here down is going to fail right now. Looking into it.

File: src/functions/views/provider.php [hunk 3]
```diff
@@ -123,6 +123,15 @@ function tribe_events_view_v2_use_period_repository() {
  * @return bool Whether Widgets v2 should load.
  */
 function tribe_events_widgets_v2_is_enabled() {
+	if ( ! tribe_events_views_v2_is_enabled() ) {
+		return false;
+	}
+
+
```
Comment (paulmskim) [inline src/functions/views/provider.php:130]:
```suggestion

```

**Review by paulmskim — COMMENTED** [review]
one small thing but looks good to me

**Review by lucatume — APPROVED** [review]

**Pull request merged**